### PR TITLE
[Suricata] Updated mappings to leverage import_mappings functionality

### LINKED ISSUE #7274: [Suricata] Missing ECS Field Mappings
## Describe the Issue

The Suricata integration does not currently have mappings for several `destination.*` and `source.*` fields and is causing conflicts in the `logs-*` Data View. 

## Screenshots
suricata_conflict

### PR DESCRIPTION
**Pull request by vinit-chauhan**
Enhancement

## What does this PR do?
This PR here, Dumps the old way of defining ECS field mappings and uses `import_mappings` functionality.  

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes #7274

## Screenshots
image

### CODE REVIEW

File: packages/suricata/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.12.1"
```
Comment (bhapas) [inline packages/suricata/changelog.yml:2]: May be `2.13.0` as this is an enhancement

File: packages/suricata/docs/README.md [hunk 2]
```diff
@@ -107,125 +107,14 @@ An example event for `eve` looks as following:
 
 | Field | Description | Type |
 |---|---|---|
-| @timestamp | Date/time when the event originated. This is the date/time extracted from the event, typically representing when the event was generated by the source. If the event source has no original timestamp, this value is typically populated by the first time the event was received by the pipeline. Required field for all events. | date |
-| cloud.account.id | The cloud account or organization id used to identify different entities in a multi-tenant environment. Examples: AWS account id, Google Cloud ORG Id, or other unique identifier. | keyword |
```
Comment (bhapas) [inline packages/suricata/docs/README.md:111]: Where can the user refer to the other mappings that are imported. Isn't there a way to show imported mappings in the docs too , Probably users are used to it?

**Comment by andrewkroh** [comment]
Until we have a solution for populating the `index.query.default_field` and the readme docs, perhaps we should leave the static ECS mappings in-place? `import_mappings: true` will function to help users that have added additional ECS fields through custom pipelines or other means.

**Comment by vinit-chauhan** [comment]
Hey @andrewkroh - 👋🏻

So, the reason I've removed those static files is because, what if we have conflicts between them? And AFAIK, there were scenarios in past where we had conflicts between them.I wonder what would happen if there's a conflict between these static files and ECS imported from `import_mappings`.
 I'll have to check whether it would consider the imported mappings or the static mapping?!

File: packages/suricata/docs/README.md (hunk 2 above)
Comment (vinit-chauhan) [reply packages/suricata/docs/README.md:111]: Hey @bhapas - Added those files back again. Would you please re-review it?

**Comment by vinit-chauhan** [comment]
It seems like the issue has been taken care of in #7657. Hence Closing the PR.

**Pull request closed without merging**